CMakeLists.txt

```diff
@@ -1,9 +1,44 @@
-PROJECT(wingetopt)
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0)
+
+project(getopt VERSION 1.0.0)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
-add_library(wingetopt STATIC src/getopt.c src/getopt.h)
+add_library(getopt STATIC src/getopt.c src/getopt.h)
```

Comment:
It is not setup for dynamic builds, would need some _dllexports for that. Could be added but considering it is only 2 functions likely not worth it.